app/helpers/motifs_helper.rb

```diff
@@ -67,4 +67,8 @@ def motif_attribute_row(legend, arg_value = nil, hint: nil, &block)
     tag.div(tag.div(legend, class: "col-md-4 text-bold text-right") +
         tag.div(value, class: "col-md-8"), class: "row")
   end
+
+  def cancel_warning_message
+    cancel_warning_message.present ? cancel_warning_message : "Êtes-vous sûr de vouloir annuler ce RDV ?"
```

Comment:
Il manque un point d’interrogation à `present?` non?
```suggestion
    cancel_warning_message.present? ? cancel_warning_message : "Êtes-vous sûr de vouloir annuler ce RDV ?"
```

---

On peut faire une clé i18n du genre `default_cancel_warning_message` plutôt que de mettre le texte en dur?